spec/webauthn/relying_party_spec.rb

```diff
@@ -21,7 +23,7 @@
   end
 
   let(:user) do
-    OpenStruct.new(id: WebAuthn.generate_user_id, name: 'John Doe', credentials: [])
+    User.new(id: WebAuthn.generate_user_id, name: 'John Doe', credentials: [])
```

Comment:
In the same line as the previous comment, I'd do:

```suggestion
    user_klass = Struct.new(:id, :name, :credentials, keyword_init: true)
    user_klass.new(id: WebAuthn.generate_user_id, name: 'John Doe', credentials: [])
```